README.md

```diff
@@ -1,29 +1,30 @@
-# Install
-
+# Dependencies
 This requires NodeJS 8.0.0 or higher which can be downloaded from [here](https://nodejs.org/en/download/) and Git which can be downloaded from [here](https://git-scm.com/).
 
-Download the zip, extract to a directory then open CMD in that directory and run `npm i` and let npm install the packages.
+For Linux users, follow these steps to install the dependencies for your distro [here](https://git-scm.com/download/linux) and [here](https://nodejs.org/en/download/package-manager/)
 
-# Setup
-Sadly some libs haven't been updated in a while to apply some fixes so you will need to edit 2 files in order for this to work. Go to the node_modules folder and look for these folders. For example node_modules/wintools
+In CMD / Terminal or whatever the equivelant is of CMD for your operating system, run: `git clone https://github.com/WalshyDev/Discord-Spotify-RPC` then, move into the newly created directory by executing `cd Discord-Spotify-RPC` and then install all packages required by Discord-Spotify-RPC: `npm install` and let NPM install the packages.
 
-## wintools
-Edit the /lib/ps.js so line 11 has a maxBuffer. Here is the new line 11 `exec('wmic process list /format:csv', {maxBuffer: 2000*1024}, function (err, stdout, stderr) {`
+# Modifying libraries
+Sadly some libraries haven't been updated in a while. This means you'll have to edit some code of the libraries otherwise it will not work.
 
-## node-spotify-webhelper
+## Modifying wintools
+Edit `node_modules/wintools/lib/ps.js` so line 11 has a maxBuffer. The new line should be like this: `exec('wmic process list /format:csv', {maxBuffer: 2000*1024}, function (err, stdout, stderr) {`
 
-You need to edit two lines here for this lib to work correctly.  
-Edit 1: line 15 the DEFAULT_PORT var, change the value to 4381.  
-Edit 2: Next go to line 158 and change the `https://%s:%d%s` to `http://%s:%d%s`
+## Modifying node-spotify-webhelper
+Windows users should skip these steps as they are unnnecessary.
```

Comment:
The port one still applies to Windows and from what I've seen the HTTP and HTTPS issue also happens sometimes on Windows. For safe measure I wouldn't say to skip these